pymesync.py

```diff
@@ -0,0 +1,60 @@
+"""
+pymesync - Python TimeSync Module
+
+Allows for interactions with the TimeSync API
+- timesync.send_time(baseurl) -- Sends time to baseurl (TimeSync)
+"""
+import errno
+import json
+import requests
+
+
+class TimeSync(object):
+
+    def __init__(self, baseurl, user, password, auth_type):
```

Comment:
This fails with a pretty good error message already:

```
Traceback (most recent call last):
  File "script.py", line 3, in <module>
    ts = pymesync.TimeSync('www.osuosl.org', 'password', 'user')
TypeError: __init__() takes exactly 5 arguments (4 given)
```

Do we need to provide more?